cmd/adapter/config_test.go

```diff
@@ -32,97 +32,71 @@ import (
 )
 
 func TestLoadCfgFromFile(t *testing.T) {
-	t.Run("test1", func(t *testing.T) {
-		// write
-		cfg := NewDefaultConfig()
-		bb, _ := json.Marshal(cfg)
-		fd, _ := writeTmpJsonFile(bb)
-		defer os.Remove(fd.Name())
-
-		// read
-		got, err := LoadCfgFromFile(fd.Name())
-		if err != nil {
-			t.Errorf("LoadCfgFromFile() error = %v", err)
-			return
-		}
-
-		// check
-		if !reflect.DeepEqual(got, cfg) {
-			t.Errorf("LoadCfgFromFile() = %v, want %v", got, cfg)
-		}
-	})
+	// write
+	cfg := NewDefaultConfig()
+	bb, _ := json.Marshal(cfg)
+	fd, _ := writeTmpJsonFile(bb)
```

Comment:
If the write fails (and it's going to be "fun" tracking this down as `os.Remove` call is also unchecked) - will the test output be reasonably clear?

Reply:
added error checking to writeTmpJsonFile()